pulp_smash/tests/puppet/api_v2/test_sync_publish.py

```diff
@@ -76,6 +82,40 @@ def _gen_repo():
         'notes': {'_repo-type': 'puppet-repo'},
     }
 
+class _CleanUpTestCaseMixin(object):
+    """Provide functions to clean up database and filesystem of Pulp"""
+
+    @classmethod
+    def reset_database(cls, server_config):
+        """Drop database and re-run migrations."""
+        cls.services = tuple((
```

Comment:
Why make this a class attribute? I don't see any advantage in filling client classes' namespaces further.